docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -0,0 +1,69 @@
+.. _2017_cause_ischemic_heart_disease:
+
+======================
+Ischemic Heart Disease
+======================
+
+Disease Description
+-------------------
+
+.. todo::
+
+   Add more information and references. In particular, find data about global prevalence and relation to disease fatal and non-fatal description.
+
+
+
+
+
+Modeling Ischemic Heart Disease in GBD 2017
+-------------------------------------------
+
+Myocardial infarction (MI) in GBD 2017
+++++++++++++++++++++++++++++++++++++++
+
+
+Modeling Strategy for Ischemic Heart Disease
+++++++++++++++++++++++++++++++++++++++++++++
+
+Cause Model Diagram
+--------------------------
+.. image:: ihd_states.png
+  :width: 150
+
+Transitions Diagram
+--------------------------
+.. image:: ihd_transitions.png
+  :width: 200
+
+Data Description
+----------------
+
+.. todo::
+
+   Add tables describing data sources for the Vivarium model.
+
+
+Model Assumptions and Limitations
+---------------------------------
+Restrictions
+------------
```

Comment:
If we want to match the heading structure of the other files, this should be +++++++ instead of ------- to make Restrictions a subsection of Model Assumptions and Limitations.

Reply:
I made this update. Thanks!